LibreTranslate.Net/LibreTranslate.cs

```diff
@@ -1,114 +1,73 @@
     {
-        En, //English
-        Ar, //Arabic
-        Zh, //Chinese
-        Fr, //French
-        De, //German
-        It, //Italian
-        Pt, //Portuguese
-        Ru, //Russian
-        Es //Spanish
-    }
-
-    public class Translate
-    {
-        public Translate()
-        {
-            wc = new WebClient();
-            LanguageList = new List<Language>();
-            Url = "https://libretranslate.com";
-            var languages = wc.DownloadString("https://libretranslate.com/languages");
-            if (languages.Contains("\"en\""))
-                LanguageList.Add(Language.En);
-            if (languages.Contains("\"ar\""))
-                LanguageList.Add(Language.Ar);
-            if (languages.Contains("\"zh\""))
-                LanguageList.Add(Language.Zh);
-            if (languages.Contains("\"fr\""))
-                LanguageList.Add(Language.Fr);
-            if (languages.Contains("\"de\""))
-                LanguageList.Add(Language.De);
-            if (languages.Contains("\"it\""))
-                LanguageList.Add(Language.It);
-            if (languages.Contains("\"pt\""))
-                LanguageList.Add(Language.Pt);
-            if (languages.Contains("\"ru\""))
-                LanguageList.Add(Language.Ru);
-            if (languages.Contains("\"es\""))
-                LanguageList.Add(Language.Es);
-        }
-
-        public Translate(string url)
+        /// <summary>
+        /// The http client
+        /// </summary>
+        private readonly HttpClient HttpClient;
+        /// <summary>
+        /// The default contructor. The default http client base uri points to https://libretranslate.com
+        /// </summary>
+        public LibreTranslate()
         {
-            wc = new WebClient();
-            LanguageList = new List<Language>();
-            Url = url;
-            var languages = wc.DownloadString($"{url}/languages");
-            if (languages.Contains("\"en\""))
-                LanguageList.Add(Language.En);
-            if (languages.Contains("\"ar\""))
-                LanguageList.Add(Language.Ar);
-            if (languages.Contains("\"zh\""))
-                LanguageList.Add(Language.Zh);
-            if (languages.Contains("\"fr\""))
-                LanguageList.Add(Language.Fr);
-            if (languages.Contains("\"de\""))
-                LanguageList.Add(Language.De);
-            if (languages.Contains("\"it\""))
-                LanguageList.Add(Language.It);
-            if (languages.Contains("\"pt\""))
-                LanguageList.Add(Language.Pt);
-            if (languages.Contains("\"ru\""))
-                LanguageList.Add(Language.Ru);
-            if (languages.Contains("\"es\""))
-                LanguageList.Add(Language.Es);
+            HttpClient = new HttpClient()
+            {
+                BaseAddress = new Uri("https://libretranslate.com"),
+                Timeout = System.Threading.Timeout.InfiniteTimeSpan
```

Comment:
Perhaps consider a more sane amount instead of infinite. This would result in a program halting forever without a warning of any kind, if the server is down, if the internet is disconnected, if there is a DNS issue, etc

Reply:
Well in this case may do sense. The reason I did this is because is on my development standards, I have worked applications with heavy processes than taker more than default. I will proceed with the changes.